### PR TITLE
Replace JSON Lint with jju

### PR DESCRIPTION
**Pull request by blakeembrey**
Found `jju` which provides a similar level of JSON parse error detail. I'll add the test case from https://github.com/raml-org/raml-js-parser/issues/151 tomorrow and see if we can fully replace JSON lint.

### CODE REVIEW

**Comment by dmartinezg** [comment]
Looks good, I would only add the JSON with the Unicode chars that was reported, as a regression test.

**Comment by blakeembrey** [comment]
@dmartinezg Thanks for the update. I've added the test case from #151 and it's passing.

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "got": "~2.4.0",
-    "json-lint": "~0.1.0",
+    "jju": "^1.2.0",
```
Comment (dmartinezg) [inline package.json]: Are we using `^` or `~` for deps? have we standardized?

I think that `~` is better

**Comment by dmartinezg** [comment]
:+1:

**Pull request merged**